awx/main/scheduler/task_manager.py

```diff
@@ -36,9 +35,8 @@
 from awx.main.utils import get_type_for_model, task_manager_bulk_reschedule, schedule_task_manager
 from awx.main.utils.common import create_partition
 from awx.main.signals import disable_activity_stream
-from awx.main.scheduler.dependency_graph import DependencyGraph
```

Comment:
Wouldn't this leave this class orphaned? No uses would remain. If you do this, you should delete that class (probably the whole module), but I might have more questions before jumping into that.

Reply:
The class does get used in the new file `awx/main/scheduler/task_manager_instances.py`